recipes/nlohmann_json/meta.yaml

```diff
@@ -0,0 +1,51 @@
+{% set name = "nlohmann_json" %}
+{% set version = "3.1.2" %}
+{% set sha256 = "e8fffa6cbdb3c15ecdff32eebf958b6c686bc188da8ad5c6489462d16f83ae54" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.tar.gz
+  url: https://github.com/nlohmann/json/archive/v{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  number: 0
+  skip: True  # [win and vc<14]
```

Comment:
Since this is header only and we need the compilers to pass some checks, this will have `run_exports` of the compilers applied. Meaning the compiler runtime libraries will be added. However we don't need those. So we can add `ignore_run_exports` here to avoid them from being added. Would suggest adding the following.

```yaml
  ignore_run_exports:
    - libcxx
    - libgcc-ng
    - libstdcxx-ng
    - vc
    - vs2015_runtime
```

Reply:
+1, with maybe a comment including what you just said :)

Reply:
Correct, it's a sub-field of `build`.

Reply:
Here's the [docs]( https://conda.io/docs/user-guide/tasks/build-packages/define-metadata.html#export-runtime-requirements ). It shows up at the end of that section.

Reply:
Note: Done below.